# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/media/systemservices/moz.build
# Commit: 703360f0b861
# Full Hash: 703360f0b8611916dfb3724eda7f35345d3f4be7
# Author: Andreas Pehrson <apehrson@mozilla.com>
# Date: 2022-12-13 04:11:09
# Regressor Bug: 1451394
# File Overlap Count: 1
# Description:
#   Bug 1451394 - Integrate with the libwebrtc camera backend for Mac. r=webrtc-reviewers,jib
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D163684
# ==============================================================================

diff -r 49002523b70d -r 703360f0b861 dom/media/systemservices/moz.build
--- a/dom/media/systemservices/moz.build	Mon Dec 12 15:47:22 2022 +0000
+++ b/dom/media/systemservices/moz.build	Mon Dec 12 15:47:22 2022 +0000
@@ -36,9 +36,15 @@
         REQUIRES_UNIFIED_BUILD = True
         UNIFIED_SOURCES += [
             "objc_video_capture/device_info.mm",
+            "objc_video_capture/device_info_avfoundation.mm",
             "objc_video_capture/device_info_objc.mm",
             "objc_video_capture/rtc_video_capture_objc.mm",
             "objc_video_capture/video_capture.mm",
+            "objc_video_capture/video_capture_avfoundation.mm",
+        ]
+        LOCAL_INCLUDES += [
+            "/third_party/libwebrtc/sdk/objc",
+            "/third_party/libwebrtc/sdk/objc/base",
         ]
         CMMFLAGS += [
             "-fobjc-arc",